File: animation/src/main/java/com/tangbing/admin/animation/MainActivity.java
package com.tangbing.admin.animation;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.audioChange).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, AudioChangeActivity.class);
                startActivity(intent);
            }
        });
        findViewById(R.id.turnPage).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, TurnPageActivity.class);
                startActivity(intent);
            }
        });
        findViewById(R.id.turnPage2).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, PageActivity.class);
                startActivity(intent);
            }
        });
        findViewById(R.id.turnPage3).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, TurntestActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: nanonanohttpddebug/src/main/java/com/tangbing/admin/nanonanohttpddebug/MainActivity.kt
package com.tangbing.admin.nanonanohttpddebug

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

class MainActivity : AppCompatActivity() {
    var mHttpServer:MyWebServer?=null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    fun startService(view:View){


        var intent2=Intent(this,Main2Activity().javaClass)
        startActivity(intent2)
    }

    override fun onResume() {
        super.onResume()
       /* mHttpServer = MyWebServer(this)
        mHttpServer?.start()*/
        var intent=Intent(this,MyService().javaClass)
         startService(intent)
    }
}